owasp-top10-2017-apps/a5/tictactoe/README.md

```diff
@@ -97,7 +100,7 @@ Now that you know the purpose of this app, what could go wrong? The following se
 
   
 
-In order to better understand how this app works, two users, `fry123` and `bender`, were created using web interface by visiting localhost:10005/create as shown below:
+In order to better understand how this attack works, two users, `fry123` and `bender`, were created using web interface by visiting _localhost:10005/create_ as shown below:
```

Comment:
We could use a link to this URL instead of an italic font. What do you think? 🙃

Reply:
Done!